pkg/apis/v1/http/handlers/dog.go

```diff
@@ -58,7 +59,7 @@ func (h DogHandler) Handle() http.HandlerFunc {
 			id, err := uuid.Parse(idString)
 			if err != nil {
 				validationErr := apperrors.NewValidationError(
-					errors.New("GET dog params failed validation"),
+					fmt.Errorf("GET dog params failed validation: %s", err),
```

Comment:
Why not take it even one step further and use the wrapping `%w` verb introduced in Go 1.13?

Reply:
Thank you!  I'll confess that I remembered that the best practices for error handling in go had changed over time, but I didn't remember the right idiom.